apmpackage/apm/data_stream/internal_metrics/elasticsearch/ingest_pipeline/default.yml

```diff
@@ -16,3 +16,7 @@ processors:
   - remove:
       field: _metric_descriptions
       ignore_missing: true
+  - remove:
+      # Removed in 8.6.0
+      field: timeseries.instance
```

Comment:
When would this stage of the ingest pipeline be triggered?

From what I understand, we're not emitting the filed from the APM Server anymore and everything is version aligned.

Reply:
This is to handle data coming from an old APM Server, that hasn't yet been upgraded. First we upgrade Kibana, which upgrades the integration package; then we upgrade Elastic Agent and therefore APM Server.

Reply:
So to be a little clearer:

> everything is version aligned

Not exactly. What we guarantee is that APM Server's version is no greater than Kibana/the integration package's version.